backend/stakepoold/rpc/rpcserver/server.go

```diff
@@ -279,3 +279,10 @@ func (s *stakepooldServer) GetStakeInfo(ctx context.Context, req *pb.GetStakeInf
 		Expired:          response.Expired,
 	}, nil
 }
+
+func (s *stakepooldServer) GetColdWalletExtPub(ctx context.Context, req *pb.GetColdWalletExtPubRequest) (*pb.GetColdWalletExtPubResponse, error) {
+	response := s.appContext.GetColdWalletExtPub()
+	return &pb.GetColdWalletExtPubResponse{
+		ColdWalletExtPub: response,
```

Comment:
Can inline this: `ColdWalletExtPub: s.appContext.GetColdWalletExtPub(),`
or as suggested in an earlier review comment `ColdWalletExtPub: s.appContext.ColdWalletExtPub,`